karate-core/src/main/java/com/intuit/karate/graal/MethodInvoker.java

```diff
@@ -50,11 +48,18 @@ public Object call(Object... args) {
         for (int i = 0; i < args.length; i++) {
             args[i] = JsValue.fromJava(args[i]);
```

Comment:
I think I made a mistake. can you confirm if `JsValue.toJava(args[i])` does the right thing

Reply:
@ptrthomas - sure I will check it out - if I recall the toJava expects a 'Value' type vs 'PolyglotMap' here, I will check.

Reply:
ah my bad, use `JsValue.unWrap()`

Reply:
actually unWrap will give us something like PolyglotMap etc in this case - I wrote a small util to fetch class instead `unWrapClass`. so it will be something like:

      Object intObj = JsValue.fromJava(args[i]);  // give us something like JsMap etc
      args[i] = JsValue.unWrap(args[i]); // unwraps and give actual argument, something like PolyglotMap etc
      types[i] = args[i] == null ? Object.class : JsValue.unWrapClass(intObj); // give matching class name.

Reply:
update: method-invoker had too many issues, so I reverted to the old way. should be stable now - there are some breaks we need to investigate: https://github.com/intuit/karate/commit/0a5a3678f2e8c678a5405a3cdaa0d42d2f14f644

Reply:
got it! will pull & merge back. Thank you.